mlflow/store/artifact/runs_artifact_repo.py

```diff
@@ -156,10 +157,20 @@ def download_artifacts(self, artifact_path, dst_path=None):
                     # Return the first model that matches the run_id and artifact_path
                     if model.source_run_id == run_id and model.name == artifact_path:
                         repo = get_artifact_repository(model.artifact_location)
-                        return repo.download_artifacts(
-                            artifact_path=".",  # root directory
-                            dst_path=dst_path,
-                        )
+                        try:
+                            return repo.download_artifacts(
+                                artifact_path=".",  # root directory
+                                dst_path=dst_path,
+                            )
+                        finally:
+                            color_warning(
+                                "`runs:/<run_id>/artifact_path` is deprecated for loading models, "
+                                "use `models:/<model_id>` instead. Alternatively, retrieve "
+                                "`model_info.model_uri` from the model_info returned by "
+                                "mlflow.xxx.log_model.",
```

Comment:
```suggestion
                                "mlflow.<flavor>.log_model.",
```

Reply:
Can we include the following in the message?

`model_info = mlflow.<flavor>.log_model(...); model = mlflow.<flavor>.load_model(model_info.model_uri)`